samples/vision_batch_annotate_files_beta.js

```diff
@@ -0,0 +1,82 @@
+/**
```

Comment:
similar feedback as above - can we drop `vision_` and `_beta` from the file name?